frontend/src/global_styles/content/_pagination.sass

```diff
@@ -28,87 +28,90 @@
 
 $pagination--font-size: 0.8125rem
 
-.pagination
+.op-pagination
   display: flex
   justify-content: space-between
   width: 100%
   min-height: 45px
 
-.pagination--pages
-  flex-grow: 2
-  flex-shrink: 2
-  margin: 10px 5px 10px 0
+  &--pages
+    flex-grow: 2
+    flex-shrink: 2
+    margin: 10px 5px 10px 0
 
-.pagination--options
-  flex-grow: 1
-  flex-shrink: 1
-  margin: 10px 0 0 5px
+  &--options
+    flex-grow: 1
+    flex-shrink: 1
+    margin: 10px 0 0 5px
 
-.pagination--items
-  list-style-type: none
-  display: flex
-  margin: 0
-  padding: 0
-  font-size: $pagination--font-size
-
-  .pagination--pages &
-    justify-content: flex-start
+  &--items
+    list-style-type: none
+    display: flex
+    margin: 0
+    padding: 0
+    font-size: $pagination--font-size
 
-  .pagination--options &
-    justify-content: flex-end
+    .op-pagination--pages &
```

Comment:
Does this need to be a child? Same for below. If you can get 1-class selectors, always try to do that.

Reply:
What do you mean with 1 class selectors? As few nestings in the sass-file as possible?

Reply:
I I understand correctly, this will produce `.op-pagination--items .op-pagination--pages .op-pagination--items` or am I missing something? in any case, it feels like the `justiyf-content` stuff should just be on `--items` directly

Reply:
No this will create `op-pagination--pages .op-pagination--items { justify-content: flex-start; }`. So in the compiled code `justify-content` is applied directly to `--items`. 
The alternative would be something this:
```
&--pages
  .op-pagination--items
    justify-content: flex-start
    
&--options
  .op-pagination--items
    justify-content: flex-end
```

I personally like it better the way it is now because then rules that belong to `--items` are still in the block of `--items` and not somewhere else.. 
Another alternative might be to introduce modifiers. Something like `--items_end` and `--items_start`. So that we are independent of `--pages` and `--options`.

Reply:
I added modifiers `_start` and `_end` so that we remove the dependency completely 👍